src/main/java/core/basesyntax/strategy/impl/BalanceHandlerImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceHandlerImpl implements OperationHandler {
```

Comment:
get rid of impls in your tests. it is not strictily required to sort tests in folders and separate "impls".